server/models/Collection.ts

```diff
@@ -390,6 +391,12 @@ class Collection extends ParanoidModel {
     });
   }
 
+  static readonly COLOR_PALETTE = colorPalette();
+
+  static randomColor() {
```

Comment:
Feels a bit out of place as a static method on `Collection` to be honest, I think we'd be okay just using `randomElement(colorPalette)` directly on the api default. It would still be very clear :)